tests/bls_tests.rs

```diff
@@ -73,37 +59,26 @@ impl AggregatingTestIO {
         let aggregate = match aggregate(&input_signatures) {
             Ok(agg) => agg,
             // handling for zero sized input and output
-            Err(_) => {
-                if self.output == String::from("~") {
-                    return true;
-                } else {
-                    return false;
-                }
-            }
+            Err(_) => return assert_eq!(self.output, "~"),
         };
         let expected_aggregate_raw = decode_hex_with_prefix(&self.output);
         let expected_aggregate = Signature::from_bytes(&expected_aggregate_raw).unwrap();
-        aggregate == expected_aggregate
+        assert_eq!(aggregate, expected_aggregate)
     }
 }
-
 #[derive(Debug, Serialize, Deserialize)]
-
 struct AggVerifyInput {
     pubkeys: Vec<String>,
     messages: Vec<String>,
     signature: String,
 }
-
 #[derive(Debug, Serialize, Deserialize)]
-
 struct AggVerifyTestIO {
     input: AggVerifyInput,
     output: bool,
 }
-
 impl AggVerifyTestIO {
-    fn aggregate_verify(&self) -> bool {
+    fn verify(&self) -> () {
```

Comment:
can drop the `()`:

```suggestion
    fn verify(&self) {
```

and other places here